Woodall_Robert/Assignments/bubble_sort.py

```diff
@@ -0,0 +1,25 @@
+import random
+import datetime
+
+def bubble_sort(list):
+	start = datetime.datetime.now()
+
+	for i in range(len(list)):
+		for j in range(len(list) - 1):
```

Comment:
If you are moving items up the list as you find them, do you have to start at the last element of the list for the inner loop?